worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
Well, some setters here take one or two values (data and theresholds or data and quantiles) and then instantiate a class (Layer, LayerWithQuantiles or LayerWithThresholds). So to have the setter we should demand users to instantiate that class and assign it to the properties. I would rather hide that in a setter. I think without the property setter we just miss the "syntax sugar". It would be cool to have it but it is not strictly needed.